### ISSUE TITLE
[MAS2.4.3] Backward navigation is not same as forward navigation while navigate on the Audio/Video clips present on the page.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1311]
**Actual Result:**
Focus directly lands on the whole audio clip while navigate in the backward navigation.
Forward navigation is not same as backward navigation while navigate on the Audio/Video clips present on the page.

**Note:** Same issue repro on all Video clip with normal mode(Tab) and scan mode.(Throughout).

**Expected Result:**
Backward navigation should be same as forward navigation while navigate on the Audio clip.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'accessibility' in the input control at the bottom of the screen and hit enter (or click on send button).
4. Page will upload, Navigate to Audio clip by using Tab key.
5. Observe that backward navigation is not same as forward navigation while navigate on the Audio/Video clips present on the page. 
 
**User Impact:**
If keyboard focus is not same in backward and forward navigation on any control then the user who is dependent on keyboard will get confuse.

**MAS Reference:**
[MAS 2.4.3 - Focus Order (05)](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9})

**Test Environment:**
**OS:** Windows 10
**OS Build**: 17134.345
**OS Version:** 1803
**Browser:** Edge

**Attachment:**
1. [MAS2.4.3_Fwd and backward navigation is not same.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2547537/MAS2.4.3_Fwd.and.backward.navigation.is.not.same.zip)
2. [MAS2.4.3_Video player scan mode.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2623224/MAS2.4.3_Video.player.scan.mode.zip)

**Comment by corinagum** [comment]
Unable to repro

**Comment by compulim** [comment]
Steps to repro:

1. <kbd>SHIFT</kbd> + <kbd>TAB</kbd> until it focus on the whole audio control, i.e. `<audio>`

Expected: <kbd>TAB</kbd> should jump to the control after `<audio>`
Actual: <kbd>TAB</kbd> jump to play button, which is the first control inside the `<audio>`

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG2.4.3
